studio/src/editors/variant-picker.js

```diff
@@ -4,6 +4,7 @@ import { html, LitElement } from 'lit';
 const VARIANTS = [
     { label: 'All', value: 'all', surface: 'all' },
     { label: 'Catalog', value: 'catalog', surface: 'acom' },
+    { label: 'Plans', value: 'plans', surface: 'acom' },
```

Comment:
plan without s? a card is a plan i think

Reply:
its plans